opslevel/datasource_opslevel_teams_all.go

```diff
@@ -0,0 +1,77 @@
+package opslevel
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+var _ datasource.DataSourceWithConfigure = &TeamDataSourcesAll{}
+
+func NewTeamDataSourcesAll() datasource.DataSource {
+	return &TeamDataSourcesAll{}
+}
+
+type TeamDataSourcesAll struct {
+	CommonDataSourceClient
+}
+
+type teamDataSourcesAllModel struct {
+	Teams []teamDataSourceModel `tfsdk:"teams"`
```

Comment:
`teamDataSourceModel` has these extra fields: `Member`, `ParentId`, `ParentAlias` that don't seem to fit with the `teams` datasource